test/acceptance/ember-cli-update-test.js

```diff
@@ -98,7 +98,7 @@ describe(function() {
       ];
     }
     if (blueprint) {
-      args.push(`-b=${blueprint}`);
```

Comment:
I believe this was also testing the short alias.

Reply:
gotcha, reverted and added comment